packages/editor-ui/src/components/CredentialCard.vue

```diff
@@ -187,6 +190,15 @@ function moveResource() {
 	&:hover {
 		box-shadow: 0 2px 8px rgba(#441c17, 0.1);
 	}
+
+	@include mixins.breakpoint('sm-and-down') {
+		flex-wrap: wrap;
+
+		:global(.n8n-card-append) {
+			width: 100%;
+			padding: 0 var(--spacing-s) var(--spacing-s) 0;
+		}
+	}
```

Comment:
Instead of having global css here and in WorkflowCard, couldn't we move `n8n-card-append` to Card.vue and use variable for the padding with a fallback? Then we'd only need to change it at a single place and without global

Reply:
Great suggestion! Improved this. Thanks